### PR TITLE
test(ingestion): Unify tests into single `process-event.test.ts`

### PR DESCRIPTION
**Pull request by Twixes**
## Problem

We had two `process-event` suites, one under `postgres` and one under `clickhouse`, both based on a common factory located under `shared`. Since Postgres ingestion is no longer a thing though, we don't need the `postgres` variant and also don't need the `clickhouse` suite to be a separate thing.

## Changes

Unifies:

- `plugin-server/tests/postgres/process-event.test.ts`
- `plugin-server/tests/clickhouse/process-event.test.ts`
- `plugin-server/tests/shared/process-event.ts`

into:
- `plugin-server/tests/shared/process-event.test.ts`

as part of #9763.

CC @macobo

### CODE REVIEW

File: plugin-server/tests/postgres/process-event.test.ts [hunk 1]
```diff
@@ -1,54 +0,0 @@
...
-            // # Test no team leakage
-            const team2 = teams[1]
-
-            const elementsHash3 = await hub!.db.createElementGroup(elements2, 3)
-            const elementGroup3 = await hub!.db.fetchElements()
-
-            // created new elements as it's different team even if the hash is the same
-            expect(elementGroup3.length).toEqual(4)
-            expect(elementsHash).toEqual(elementsHash2)
-            expect(elementsHash).toEqual(elementsHash3)
```
Comment (tiina303) [inline plugin-server/tests/postgres/process-event.test.ts:51]: don't we lose coverage here or do we not care about this elements group thing anymore?

**Review by tiina303 — APPROVED** [review]
Nice work :)

File: plugin-server/tests/postgres/process-event.test.ts (hunk 1 above)
Comment (Twixes) [reply plugin-server/tests/postgres/process-event.test.ts:51]: This only applies (applied) to Postgres. In ClickHouse the elements are stored inline with events (as `elements_chain`)

File: plugin-server/tests/shared/process-event.test.ts [hunk 2]
```diff
@@ -95,12 +73,17 @@ export const getEventsByPerson = async (hub: Hub): Promise<EventsByPerson[]> =>
...
+const TEST_CONFIG: Partial<PluginsServerConfig> = {
+    PLUGINS_CELERY_QUEUE: 'test-plugins-celery-queue',
+    CELERY_DEFAULT_QUEUE: 'test-celery-default-queue',
+    LOG_LEVEL: LogLevel.Log,
+    KAFKA_ENABLED: true,
+    KAFKA_HOSTS: process.env.KAFKA_HOSTS || 'kafka:9092',
+    KAFKA_CONSUMPTION_TOPIC: KAFKA_EVENTS_PLUGIN_INGESTION,
+}
+
+export const createProcessEventTests = (includeNewPropertiesUpdatesTests: boolean): ReturnWithHub => {
```
[the remark this answers is not in the document]
Comment (Twixes) [reply plugin-server/tests/shared/process-event.test.ts:85]: 👍 Cool, I'll yeet that too then

**Pull request merged**